### DISCUSSION TITLE
Type or namespace of MyModal in Modal.Show<MyModal> cannot be found

### DISCUSSION THREAD

**Discussion by Kurti123** [Blazored/Modal#318] [General]
Sometimes in VS 2019 I stumble across the error that the reference to my modal cannot be found when I call the Modal.Show () method. In earlier methods of VS 2019, this was an error that was almost guaranteed but it improved with 16.10 and only occurs from time to time. 

Am I the only person experiencing this error? I think that's rather a VS problem but I'm curious if anybody else is facing this issue. 

![BlazoredModal](https://user-images.githubusercontent.com/24918126/121475432-afcb2b00-c9c5-11eb-8ad7-ff03a3ee32b6.png)

**Comment by larsk2009** [comment]
Does it still compile or not? I have had this before as well and the solution is to compile. That does work, and after that the issue is gone.

I think this issue has nothing to do with this project, but is just showing that visual studio does not do perfect syntax highlighting for Blazor yet.

**Reply by Kurti123** [reply]
It compiles but the error warning won't go away.

**Reply by chrissainty** [reply]
Hi @Kurti123, as @larsk2009 mentioned, this is a Visual Studio issue/bug. It happens on and off for me on lots of different Blazor projects. As long as the app is building the error is irrelevant.
